app/controllers/admin/base_controller.rb

```diff
@@ -20,16 +20,28 @@ def should_count?
         controller_name === "services" || "requests"
     end
 
-    def set_counts
-        @all_count = Service.kept.count
-        @ofsted_count = Service.kept.ofsted_registered.count
-        @pending_count = Service.kept.where(approved: nil).count
-        @archived_count = Service.discarded.count
-        if APP_CONFIG["directories"].present?
-            @directories_counts = {}
+    def set_counts        
+          @service_counts_all = {
+            all: Service.kept.count,
+            ofsted: Service.kept.ofsted_registered.count,
+            pending: Service.kept.where(approved: nil).count,
+            archived: Service.discarded.count
+          }
+          @service_counts = {}
+          @service_counts[:all] = @service_counts_all
+      
+          if APP_CONFIG["directories"].present?
```

Comment:
If the directories config isn't there there these extra counts are never called so we shouldn't need an else? the `if APP_CONFIG["directories"].present?` check is used everywhere. No errors occur when its not present. 

Though I have noticed the divider icon is in the wrong place when that directory config isn't there - so I've fixed that. 😂 

![image](https://user-images.githubusercontent.com/649148/131487469-369cda77-77de-41da-a2d3-cd813ae8cbb5.png)
![image](https://user-images.githubusercontent.com/649148/131487487-70a43568-dbb5-4c01-9eec-c68960d89b92.png)
![image](https://user-images.githubusercontent.com/649148/131487517-e719862b-a667-4431-b9a6-8718c314349f.png)

Reply:
Ah strange, thought I got an error when I had a click around the other day, without the instance set. Maybe you've fixed since or something else was going on. Ah yeh I noticed the divider thing too, thanks for fixing!